documentation/asciidoc/topics/proc_securing_custom_configuration.adoc

```diff
@@ -0,0 +1,39 @@
+[id='secure_credentials_{context}']
+= Securing custom {brandname} configuration
+
+[role="_abstract"]
+Define and store custom {brandname} configuration that requires credentials securely.
+To protect sensitive text strings such as passwords, add the entries in a credential store rather than directly in the `Infinispan` CR.
```

Comment:
Credentials would be stored in the server configuration `xml|json|yaml`, not the Infinispan CR itself.